Document/0x06i-Testing-Code-Quality-and-Build-Settings.md

```diff
@@ -347,6 +347,34 @@ func dosomething(argumentx:TypeX) throws {
 
 - Use the `try!` expression to assert that the error won't occur.
 
+
+- Handle the error as a `Result` return:
+
+```swift
+enum ErrorType: Error {
+    case typeOne
+    case typeTwo
+}
+
+func functionWithResult(param: String?) -> Result<String, ErrorType> {
+    guard let value = param else {
+        return .failure(.typeOne)
+    }
+    return .success(value)
+}
+
+func callResultFunction() {
+    let result = functionWithResult(param: "OWASP")
+
+    switch result {
+    case let .success(value):
+        print(value)
+    case let .failure(error):
+        print("Error: \(error)")
+    }
+}
+```
+
 #### Static Analysis
```

Comment:
how about giving one snippet of a generic code and one snippet of a code actually being used in day to day practice like error/exception handling while making network calls ?

Reply:
Maybe we should be careful with that ^^. But having a link to a proper exception handling example in swift 5 with TLS /json could be great!

Reply:
Thanks! I updated the references for the `Result` type and added a network/JSON decoding example.